docs/_docs/key-value-api/read-repair.adoc

```diff
@@ -0,0 +1,99 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+= Read Repair
+
+WARNING: Experimental API
+
+[WARNING]
+====
+[discrete]
+=== Limitations
+A consistency check is incompatible with the following cache configurations:
+
+* Caches without backups.
+* Near caches.
+* Caches that use "read-through" mode.
+====
+
+
+`Read Repair` refers to a technique of repairing inconsistencies between primary and backup copies of data during normal read operations. When a specific key (or keys) is read by a user operation, Ignite checks the values for the given key in all backup copies.
+
+The `Read Repair` mode is designed to maintain consistency. However, read operations become {tilde}2 times more costly because backup copies are checked in addition to the primary copy. It is generally not advisable to use this mode all the time, but rather on a once-in-a-while basis.
+
+To enable the `Read Repair` mode, obtain an instance of the cache that enables Read Repair reads as follows:
```

Comment:
Sounds too prescriptive

`it is advisable to obtain` sounds much friendlier.

Reply:
It's not advisable, it is necessary